internal/service/systemd.go

```diff
@@ -164,7 +163,7 @@ func (s *systemd) Disable() error {
 }
 
 func (s *systemd) serviceName(serviceName string) string {
-	return s.servicePrefix + serviceName + ServiceSuffix
+	return ServicePrefix + serviceName + ServiceSuffix
```

Comment:
That breaks `node_exporter` service control - it requires blank prefix.

Reply:
Done.